api/app/Models/PoolCandidate.php

```diff
@@ -566,8 +566,8 @@ public function createSnapshot()
             'poolCandidates.educationRequirementEducationExperiences.skills',
             'poolCandidates.educationRequirementPersonalExperiences.skills',
             'poolCandidates.educationRequirementWorkExperiences.skills',
-            'poolCandidates.screeningQuestionResponses',
-            'poolCandidates.screeningQuestionResponses.screeningQuestion',
+            'poolCandidates.generalQuestionResponses',
```

Comment:
Ooh, this is sticky - we're not going to be able to render old snapshots with this change, right?  Can we be clever and find the responses in the snapshot under the old property name, too?

Reply:
Ooh yeah good point, sticky indeed. 
I took a swing at it and with some testing it did appear to be working for me trying it out
https://github.com/GCTC-NTGC/gc-digital-talent/pull/9070/commits/2e3560f242c03d077460af966116784cd276d9dc

Reply:
Nice, this is working well for me.  I'm going to ask how long we have to keep these kinds of workarounds around.